src/rpm_ostree/cli_status.rs

```diff
@@ -62,6 +71,7 @@ pub struct Status {
 pub struct Deployment {
     booted: bool,
     container_image_reference: Option<String>,
+    container_image_reference_digest: Option<String>,
     custom_origin: Option<CustomOrigin>,
```

Comment:
Basically, I'd just nuke this field entirely and all the related bits and stop caring about it.